churaverse-plugins-server/src/synchroBreakPlugin/controller/socketController.ts

```diff
@@ -38,6 +41,8 @@ export class SocketController extends BaseSocketController<IMainScene> {
     ev.messageRegister.registerMessage('nyokkiTurnEnd', NyokkiTurnEndMessage, 'allClients')
     ev.messageRegister.registerMessage('nyokkiTurnStart', NyokkiTurnStartMessage, 'allClients')
     ev.messageRegister.registerMessage('updatePlayersCoin', UpdatePlayersCoinMessage, 'allClients')
+    ev.messageRegister.registerMessage('nyokkiResult', NyokkiResultMessage, 'allClients')
+    ev.messageRegister.registerMessage('nyokkiGameEnd', NyokkiGameEndMessage, 'allClients')
```

Comment:
シンクロブレイクの終了メッセージは、server側でのみ必要な情報だと思うので、送信先もそのようにした方がいいと思います
```suggestion
    ev.messageRegister.registerMessage('nyokkiGameEnd', NyokkiGameEndMessage, 'onlyServer')
```